react-native-libraries.json

```diff
@@ -6170,5 +6170,16 @@
     "windows": false,
     "macos": false,
     "unmaintained": false
+  },
+  {
+    "githubUrl": "https://github.com/microsoft/fluentui-react-native",
+    "npmPkg": "fluentui-react-native",
+    "ios": true,
+    "android": true,
```

Comment:
```suggestion
    "android": false,
```

Reply:
i change your suggestion thank you @brentvatne !.